Makefile

```
@@ -19,7 +19,8 @@ LOCAL_GOOS ?= $(shell go env GOOS)
 LOCAL_GOARCH ?= $(shell go env GOARCH)
 
 REPONAME ?= signoz
-DOCKER_TAG ?= latest
+# use branch name as docker tag if not provided
+DOCKER_TAG ?= $(shell git rev-parse --abbrev-ref HEAD)
```

We will hit the following error in case of `/` or other special 

```
ERROR: invalid tag "signoz/frontend:chore/test": invalid reference format
```


```suggestion
DOCKER_TAG ?= $(subst v,,$(BUILD_VERSION))
```